Shoko.Server/Providers/AniDB/UDP/AniDBSocketHandler.cs

```diff
@@ -22,6 +22,8 @@ public class AniDBSocketHandler : IAniDBSocketHandler
     private readonly SemaphoreSlim _semaphore = new SemaphoreSlim(1, 1);
     private bool Locked { get; set; }
     public bool IsLocked => Locked;
+    private static int SendTimeoutMs => 30000; // 30 seconds
+    private static int ReceiveTimeoutMs => 30000; // 30 seconds
```

Comment:
For future reference, what would be better is using TimeSpan objects to specify this, then you can use [TimeSpan.TotalMilliseconds](https://learn.microsoft.com/en-us/dotnet/api/system.timespan.totalmilliseconds?view=net-8.0)